ceph_iscsi_config/gateway.py

```diff
@@ -231,6 +231,14 @@ def clear_config(self):
             self.error = True
             self.error_msg = "Unable to delete target - {}".format(err)
 
+    def set_noacl(self, config):
+        target_config = config.config["targets"][self.iqn]
+        for tpg in self.tpg_list:
+            if target_config['noacl']:
+                tpg.set_attribute('generate_node_acls', 1)
```

Comment:
I'm not familiar w/ the configfs layout, but setting ```generate_node_acls``` to 1 will disable ACLs? Seems backwards to me.

Reply:
Yes, the attribute name may be confusing, but it disabled the ACL authentication - https://wiki.alpinelinux.org/wiki/Linux_iSCSI_Target_(TCM)